schemas/related.yml

```
@@ -70,3 +70,15 @@
         identifiers include FQDNs, domain names, workstation names, or aliases.
       normalize:
         - array
+
+    - name: entity
+      level: extended
+      type: keyword
+      short: All the entity identifiers
+      description: >
+        All the entity identifiers related to the document. If the document
+        contains multiple entities, identifiers belonging to different entities
+        will be present. Example identifiers include Cloud Resource Ids, ARNs, email
```

```suggestion
        will be present. Example identifiers include cloud resource IDs, ARNs, email
```

I _think_ ECS uses "ID" as shorthand for identifier. And don't think we need to capitalize "Cloud Resource".

---

I agree. I've seen ID in multiple spots too. I'll commit the suggestion (and the impact it has on auto generated files)